recipe/bld.bat

```diff
@@ -1,16 +1,20 @@
-mkdir build && cd build
+@echo on
 
-cmake -G "NMake Makefiles"^
-    -DCMAKE_BUILD_TYPE=Release^
-    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%^
-    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX%^
-    -DCMAKE_INSTALL_LIBDIR=lib^
-    %SRC_DIR%
+cmake %SRC_DIR% ^
```

Comment:
Note that this will use the Visual Studio genereator, that being a multiple config generator is frequently not tested in depth by project. I tipically use `-GNinja` also on Windows as it works well in my experience.

Reply:
Yeah, I intentionally changed it because I would prefer using a more default generator that people might use, which I believe is the Visual Studio generator. However, I'm okay with using Ninja.